x-pack/legacy/plugins/siem/public/components/ml_popover/api.tsx

```diff
@@ -97,18 +100,21 @@ export const setupMlJob = async ({
  * Starts the given dataFeedIds
  *
  * @param datafeedIds
+ * @param start
  * @param headers
  */
 export const startDatafeeds = async (
   datafeedIds: string[],
-  headers: Record<string, string | undefined>
+  headers: Record<string, string | undefined>,
+  start = 0
 ): Promise<StartDatafeedResponse> => {
   try {
     const response = await fetch('/api/ml/jobs/force_start_datafeeds', {
       method: 'POST',
       credentials: 'same-origin',
       body: JSON.stringify({
         datafeedIds,
+        ...(start !== 0 && { start }),
```

Comment:
That's interesting way to do a spread. I tried it out in my console and it looks to work correctly.

Nothing wrong with it, I just haven't seen one like that before. Works well 👍